### PR TITLE
Update contingency plan for staffing and internal resources

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:

- Recovery of leadership credentials
- Minimum staffing and stabilization
- Jumpboxes

Technical details are in runbooks to facilitate development and updates of required steps

## security considerations

Keeping some details private to sidestep social engineering.

### CODE REVIEW

**Review by bengerman13 — DISMISSED** [review]
LGTM

**Comment by RonWilliams** [comment]
Eddie and I have some changes before moving this to prod. Please hold off on merge for now.

**Comment by pburkholder** [comment]
@RonWilliams  I thought I'd already tagged you and Eddie, as indeed I need your feedback before merging. Thanks for chiming in. Do you want to discuss today during the CP training?

**Comment by pburkholder** [comment]
@eddietejeda @RonWilliams What changes would you like here before merging to prod?

File: _docs/ops/contingency-plan.md [hunk 1]
```diff
@@ -86,14 +86,56 @@ The Incident Commander declares that recovery efforts are complete and notifies
...
+
+The director and deputy director own administrative credentials critical to long-term
+platform operation and customer support. The 
+[recovery process for accessing those credentials](https://github.com/cloud-gov/internal-docs/blob/master/runbooks/contingency.md#leadership-credentials)
+ are non-public (to avoid potential social engineering). These credentials would be delegated to a GSA interim cloud.gov system owner if leadership is
+ abruptly unavailable for 3 or more working days, or if COOP plan is otherwise in effect.
+
+### Engineer staffing
+
+If the number of available platform engineers is
```
Comment (RonWilliams) [inline _docs/ops/contingency-plan.md]: The language in this section is a concern. The details are too prescriptive and usurps the authority of the deputy director, director, and TTS leadership; specifically, leadership must maintain the authority determine staffing requirements, reaction to limited staffing, and tasking of staff.

**Review by RonWilliams — DISMISSED** [review]
Language related to engineer staffing should be less prescriptive.

File: _docs/ops/contingency-plan.md (hunk 1 above)
Comment (pburkholder) [reply _docs/ops/contingency-plan.md]: How about prefacing it with:

> In the event that the Director, Deputy Director, or TTS leadership are unavailable to make an independent determination, then if the number of engineers _etc,, etc_...

**Comment by pburkholder** [comment]
@RonWilliams Does my suggested wording change work for your needs? That is, prefacing the "stand-down" guidance with:

 >  In the event that the Director, Deputy Director, or TTS leadership are unavailable to make an independent determination, then if the number of engineers etc,, etc...

**Comment by RonWilliams** [comment]
@pburkholder I'd like to remove the prescriptive language entirely, thus allowing for determination and decisions to be made using the standard chain of command.

**Comment by pburkholder** [comment]
Comment from JAB TR-R, who would like to see more detail and ensure it's tracked in a POA&M (new or existing)

> It seems areas of improvement needed in order to be prepared for an event that would result in a major loss of personnel are listed at https://github.com/cloud-gov/cg-site/pull/1608 . Is that correct? If so, could you elaborate a bit more on what the specific issue is for each of these areas:
> -Recovery of leadership credentials
> -Minimum staffing and stabilization
> -Jumpboxes

File: _docs/ops/contingency-plan.md [hunk 2]
```diff
@@ -86,14 +86,53 @@ The Incident Commander declares that recovery efforts are complete and notifies
 
 Next, the team schedules an internal postmortem to discuss the event. This is the same as the [security incident retrospective process]({{ site.baseurl }}{% link _docs/ops/security-ir.md %}#initiate). Then we write a public postmortem and post it on StatusPage for users. We should also discuss with our JAB representatives whether they need additional information.
 
+## Internal dependencies
+
+Continued operation of cloud.gov relies on our people, but also on access to internal diagnostics/repair through our `jumpbox`. We also have
+internal GSA dependencies that may be impacted if a GSA COOP (Continuity of Operations Plan) is in effect. Resources for the GSA/TTS COOP plan include:
```
Comment (afeld) [inline _docs/ops/contingency-plan.md]: Minor nitpick on wording.

```suggestion
internal GSA dependencies that may be impacted if a GSA COOP (Continuity of Operations) event is in effect. Resources for the GSA/TTS COOP plan include:
```

**Review by afeld — COMMENTED** [review]
Hey hey, jumping in here per request from @eddietejeda. Make sure to check in with Diana Cameron on this, as she's TTS' Emergency Coordinator. We just need to make sure this information is up to date in the COOP Playbook.

**Comment by pburkholder** [comment]
The CP-2 control and the guidance from NIST for information system contingency plans, (https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-34r1.pdf) is that DRP and COOP are out of scope for an ISCP (information system contingency plan). So we're keeping the references (as I don't know where else to look for them), but moving the content to our internal docs as advisory resources. 

@afeld So I don't think we'll need to loop in Diana Cameron for this change, but we do need to coordinate with her down the line.

**Review by onelittlebecca — APPROVED** [review]

**Comment by onelittlebecca** [comment]
Hey, I approved the PR but based on the comments and requested changes from @RonWilliams I think the merging is blocked. Do you need help cleaning up the feedback from Ron and/or Aidan? Do we need to contact anyone else for approval?

**Pull request merged**